app/controllers/ems_storage_controller.rb

```diff
@@ -31,7 +31,6 @@ def check_generic_rbac
 
   def type_feature_role_check
     return true unless TYPE_CHECK_SHOW_IDENTIFIERS.include?("#{controller_name}_#{action_name}") && respond_to?(:feature_role)
-
```

Comment:
Typically we want a blank line after a guard clause, actually I'm surprised miq-bot/rubocop didn't warn about this